src/Resources/ContactProperties.php

```diff
@@ -131,7 +154,7 @@ public function getGroups($includeProperties = false)
      */
     public function createGroup(array $group)
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/groups';
+        $endpoint = "https://api.hubapi.com/contacts/v2/groups";
```

Comment:
Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)